qnapstats/qnap_stats.py

```diff
@@ -193,7 +193,7 @@ def get_smart_disk_health(self):
                   "capacity": disk["Capacity"],
                   "model": disk["Model"],
                   "serial": disk["Serial"],
-                  "type": "hdd" if int(disk["hd_is_ssd"]) == 0 else "ssd",
+                  "type": "ssd" if "hd_is_ssd" in disk else "hdd",
```

Comment:
On my TS-451, which only has HDDs, I always get an `hd_is_ssd` entry in the XML:

```xml
<hd_is_ssd><![CDATA[0]]></hd_is_ssd>
```

Can you describe how your device behaves?  Do you have both HDDs and SSDs, and does this entry only appear for your SSDs?